crates/ruff_linter/resources/test/fixtures/flake8_pytest_style/PT017.py

```diff
@@ -12,6 +12,15 @@ def test_ok():
     assert e.value.message
 
 
+def allow_exception_in_message():
+    try:
+        something()
+    except Exception as e:
+        something_else()
+
+    assert 1 == 1, e
```

Comment:
This isn't a great example, because it's not valid Python. The `e` binding doesn't last outside the `except` block:

```pycon
>>> def allow_exception_in_message():
...     try:
...         something()
...     except Exception as e:
...         something_else()
...     print(e)
... 
>>> def something(): raise Exception
... 
>>> def something_else(): pass
... 
>>> allow_exception_in_message()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<stdin>", line 6, in allow_exception_in_message
UnboundLocalError: cannot access local variable 'e' where it is not associated with a value
```